README.md

```diff
@@ -15,6 +15,12 @@ Repository that contains various tools for maintaining cdnjs.
 - `BOT_BASE_PATH`: cdnjs home
 - `SENTRY_DSN` sentry data source name (DSN)
 
+## Dependencies
+
+- [brotli](https://github.com/google/brotli)
+- [gzip](https://www.gzip.org/)
+- [zopflipng](https://github.com/google/zopfli)
+
```

Comment:
```suggestion
- [jpegoptim](https://github.com/tjko/jpegoptim)
```
or
```suggestion
- [jpegoptim](https://www.kokkonen.net/tjko/projects.html)
```

I'm not sure if we should mention: `uglify-es`, `uglify-js` and `cleancss`. We should at least mention that you are expected to run `npm install` in the `tools` directory (I'm not sure why that approach was chosen, I would prefer just to run `npm install uglify-js -g` etc).